sub-packages/bionemo-scdl/tests/bionemo/scdl/index/test_row_feature_index.py

```diff
@@ -113,6 +176,28 @@ def test_feature_index_internals_on_append(create_first_RowFeatureIndex):
     assert label == "MY_DATAFRAME"
 
 
+def test_feature_index_internals_on_append_same_features(create_first_RowFeatureIndex):
+    one_feats = {"feature_name": np.array(["FF", "GG", "HH"]), "feature_int": np.array([1, 2, 3])}
+    create_first_RowFeatureIndex.concat(create_first_RowFeatureIndex)
+    # append(8, two_feats, len(two_feats["feature_name"]), "MY_DATAFRAME")
+    assert len(create_first_RowFeatureIndex) == 1
+    assert create_first_RowFeatureIndex.number_vars_at_row(1) == 3
+    assert create_first_RowFeatureIndex.number_vars_at_row(13) == 3
+    assert create_first_RowFeatureIndex.number_vars_at_row(19) == 3
+    assert create_first_RowFeatureIndex.number_vars_at_row(2) == 3
```

Comment:
i think im still missing something, but isnt the key thing checking that theyre the same `RowFeatureIndex`? or (thinking outloud) was the behavior before something like:

[FF, GG, HH, FF, GG, HH], 0=>6

and now its
[FF,GG,HH] => 3 
?

Reply:
They are all the same. I can check this with the for loop.

Reply:
Before, it would have been 3 at every position. But it would have been stored as [[FF,GG,HH] , [FF,GG,HH] ]